src/statement/statement.njk

```diff
@@ -43,66 +54,72 @@
     </div>
   </div>
 
-  <style>
-    .sym { float: left; }
-  </style>
-  <table class="govuk-table">
-    <thead class="govuk-table__head">
-      <tr class="govuk-table__row">
-        <th class="govuk-table__header" scope="col">Name</th>
-        <th class="govuk-table__header" scope="col">Plan</th>
-        <th class="govuk-table__header" scope="col" style="text-align: right">Ex VAT</th>
-        <th class="govuk-table__header" scope="col" style="text-align: right">Inc VAT</th>
-      </tr>
-    </thead>
-    <tbody class="govuk-table__body">
-      {% for guid, items in items | groupby('spaceGUID') %}
+  {% if items | length %}
+    {% if isCurrentMonth %}
+      <p class="paas-table-notification">
+        This statement show up to date provitional usage data for current month.
```

Comment:
`s/show/shows`

Reply:
Yeah, also provisional.  Should turn on your spellcheck.